<?xml version="1.0"?>
<launch>
  <node name="vicon" pkg="vicon_ros" type="vicon_node" output="screen">
    <param name="hostName" value="vicon" />
    <param name="enableLatencyTopic" value="false" />
    <param name="enablePointCloudTopic" value="true" />
    <param name="enableTfBroadcast" value="false" />
    <param name="enablePosesTopic" value="false" />
  </node>

  <node name="configuration" pkg="object_tracker" type="configuration_node" output="screen">
  </node>

</launch>
